### ISSUE TITLE
timeout

### ISSUE THREAD

**Issue by Kat-CeDe** [kennymc-c/ucr2-integration-requests#2]
Moin from Germany again,
you have planned to allow changes of timeout during setup of the integration but this would be valid for all commands then. What about another separator like |1000 to set timeout to 1000 for this command and overrule the default.

p.s. Moin means good morning in nothern Germany:-)

Ralf

**Comment by kennymc-c** [comment]
I thought about that too but what would be the benefit? If one or more servers need a longer timeout other servers are not affected by this if they response earlier. You just get a slower response if the faster servers are not available. Also I don't want to introduce to many different parameters that can be combined as this makes it more complicated to parse the input. What if the payload data also contains the timeout separator character?

**Comment by Kat-CeDe** [comment]
The idea with the timeout came to me with my 1st experiment with a post. I send posts to switch on/off my lights in a room. When I say switch it is not correct I always use the transition (dimming) of at least 2 seconds. Because I got a timeout during this test I assumed it must have something to do with the timeout. I made another test yesterday with a transition of 6 seconds and got not timeout. I think timeout is not important until a timeout situation can be reproduced.
